app.js

```diff
@@ -47,8 +47,11 @@ function clientLoaded (client) {
  * @return {boolean} - if the error is fatal or not
  */
 function isFatal(err) {
-  return !(err.name === 'InboundHungup' || err.name === 'DialedHungup' ||
-      err.name === 'HangupFailure' || err.name === 'NoStations');
+  return !(err.name === 'DialedHungup' || err.name === 'HangupFailure' ||
```

Comment:
Since you have used lodash already, you should look into storing an array of error names that are considered fatal and use _.contains to determine if the error passed in to isFatal belongs to that list.